app/assets/stylesheets/admins/custom.scss

```
@@ -6,3 +8,34 @@
   position: absolute;
   right: 0;
 }
+
+.field_with_errors .form-group {
+  border-color: $red;
+  padding-right: calc(1.5em + 0.75rem);
+  background-repeat: no-repeat;
+  content: "*";
+  color: $red;;
+}
+
+.error-feedback {
+  width: 100%;
+  margin-top: 0.25rem;
+  font-size: 80%;
+  color: $red;
+}
+
+.field_with_errors .form-control {
+  border-color: $red;
+  padding-right: calc(1.5em + 0.75rem);
+  background-repeat: no-repeat;
+  background-position: right calc(0.375em + 0.1875rem) center;
+  background-size: calc(0.75em + 0.375rem) calc(0.75em + 0.375rem);
+}
+
+.field_with_errors .checkout__input {
+  border-color: $red;
+  padding-right: calc(1.5em + 0.75rem);
+  background-repeat: no-repeat;
+  background-position: right calc(0.375em + 0.1875rem) center;
+  background-size: calc(0.75em + 0.375rem) calc(0.75em + 0.375rem);
+}
```

viết theo cú pháp scss cho ngắn gọn, dễ theo dõi, cho cả phần .form-group vào đây 
```suggestion
.field_with_errors {
  .form-control, .checkout__input {
    border-color: $red;
    padding-right: calc(1.5em + 0.75rem);
    background-repeat: no-repeat;
    background-position: right calc(0.375em + 0.1875rem) center;
    background-size: calc(0.75em + 0.375rem) calc(0.75em + 0.375rem);
  }
}
```

---

Oki anh giai ơi